src/clarity-angular/datagrid/datagrid-column.ts

```diff
@@ -43,10 +44,16 @@ export class DatagridColumn extends DatagridFilterRegistrar<DatagridStringFilter
         super(filters);
         this._sortSubscription = _sort.change.subscribe(sort => {
             // We're only listening to make sure we emit an event when the column goes from sorted to unsorted
```

Comment:
No, we're emitting it in the sort method itself. This is only here for the case where another column becomes sorted, which automatically de-activates this one.